admin/language/en-gb/extension/payment/iyzico_checkout_form.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+// Heading
+$_['heading_title'] = 'Iyzico Checkout Form';
+
+// Text 
+$_['text_link'] = 'Register to become iyzico Merchant';
+$_['text_payment'] = 'Payment';
+$_['text_success'] = 'iyzico checkout settings updated';
+$_['text_iyzico_checkout_form'] = '<a href="http://www.kahvedigital.com/" target="_blank"><img src="view/image/payment/iyzico_checkout_form.png" alt="iyzico" title="iyzico"/></a>';
+$_['text_iyzico_checkout_form_info'] = 'You can get your API ID and Secret Key values from <a href="https://merchant.iyzipay.com/settings" target="_blank">https://merchant.iyzipay.com/settings</a>';
+// Entry
+$_['entry_api_id_live'] = 'Live Api Id';
+$_['entry_secret_key_live'] = 'Live Secret Key';
+$_['entry_order_status'] = 'Order Status';
+$_['entry_status'] = 'Status';
+$_['entry_sort_order'] = 'Sort Order';
+$_['entry_error_version'] = 'There is a new version. Click here to update.';
+$_['entry_error_version_updated'] = 'Update Error! go to iyzico version page <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">Version page</a> ';
```

Comment:
"Update Error! <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">Click here </a> to manually update the plugin.